src/api/amm/poolTokensRate.js

```diff
@@ -0,0 +1,17 @@
+const { Joi } = require('dvf-utils')
+const get = require('../../lib/dvf/get-generic')
+const validateWithJoi = require('../../lib/validators/validateWithJoi')
+
+const schema = Joi.object({
+  poolName: Joi.string()
+})
+
+const validateInputs = validateWithJoi(schema)('INVALID_METHOD_ARGUMENT')({
+  context: 'poolTokensRate'
+})
+
+module.exports = async (dvf, data) => {
+  const { poolName } = validateInputs(data)
+  const endpoint = `/v1/trading/amm/poolTokensRate/${poolName}`
```

Comment:
This could be an expensive endpoint to call to just get the rates especially if it is being polled, but it could be fine for now. ideally to just get the rate, we should read it from the db (we can repurpose this same endpoint to just read it from db and only for submitting orders use the AMM engine to get staking info)